### ISSUE TITLE
Privoxy not working for the newest version

### ISSUE THREAD

**Issue by ummecasino** [jshridha/docker-nzbgetvpn#12]
After upgrading to the newest version of the nzbgetvpn image privoxy doesn't seem to work/start and I wasn't able to use the proxy in FF. After downgrading to the prior version everything worked again. I took a process overview of the running processes and the privoxy process seems missing.

On v20.0_2.3.9-103:

```bash
sh-4.4# ps aux
USER       PID %CPU %MEM    VSZ   RSS TTY      STAT START   TIME COMMAND
root         1  0.0  0.0   4196   660 ?        Ss   12:18   0:00 /usr/bin/tini -- /bin/bash /root/init.sh
root         6  0.0  0.1  98864 20108 ?        S    12:18   0:00 /usr/bin/python2 /usr/bin/supervisord -c /etc/supervisor.conf -n
root       112  0.0  0.0  15964  3420 ?        S    12:18   0:00 /bin/bash /root/start.sh
nobody     113  0.0  0.0  15844  3416 ?        S    12:18   0:00 /bin/bash /home/nobody/nzbget.sh
nobody     114  0.0  0.0  15844  3412 ?        S    12:18   0:00 /bin/bash /home/nobody/privoxy.sh
root       265  0.0  0.0  85000  7780 ?        Ss   12:18   0:00 /usr/bin/openvpn --cd /config/openvpn --config /config/openvpn/openvpn.ovpn --daemon --dev tun0 --remote de-frankfurt.privateinternetaccess.com 1198 --proto udp --reneg-sec
nobody    1371  0.3  0.0 286840  3556 ?        Ssl  12:18   0:07 /usr/bin/nzbget -D -c /config/nzbget.conf
nobody    1383  0.0  0.0 2092836 4632 ?        Sl   12:18   0:00 /usr/bin/privoxy --no-daemon /config/privoxy/config
root      2858  0.0  0.0   5840   696 ?        S    12:52   0:00 sleep 30s
nobody    2872  0.0  0.0   5840   668 ?        S    12:52   0:00 sleep 30s
root      2873  0.0  0.0  15976  3788 pts/0    Ss   12:53   0:00 sh
root      2878  0.0  0.0  34712  3208 pts/0    R+   12:53   0:00 ps aux
```

On latest:

```sh
sh-5.0# ps aux
USER       PID %CPU %MEM    VSZ   RSS TTY      STAT START   TIME COMMAND
root         1  0.0  0.0   2300   764 ?        Ss   13:05   0:00 /usr/bin/tini -- /bin/bash /root/init.sh
root         6  0.6  0.1  30012 22420 ?        S    13:05   0:00 /usr/bin/python2 /usr/bin/supervisord -c /etc/supervisor.conf -n
root       138  0.0  0.0   7324  3708 ?        S    13:05   0:00 /bin/bash /root/start.sh
nobody     139  0.0  0.0   7192  3544 ?        S    13:05   0:00 /bin/bash /home/nobody/nzbget.sh
root       300  0.0  0.0  23828  5732 ?        Ss   13:05   0:00 /usr/bin/openvpn --daemon --reneg-sec 0 --mute-replay-warnings --auth-nocache --setenv VPN_PROV pia --setenv DEBUG  --setenv VPN_DEVICE_TYPE tun0 --setenv VPN_REMOTE de-fra
nobody     563  0.0  0.0 219304  3176 ?        Ssl  13:05   0:00 /usr/bin/nzbget -D -c /config/nzbget.conf
root       587  0.0  0.0   5280   760 ?        S    13:06   0:00 sleep 30s
nobody     598  0.0  0.0   5280   696 ?        S    13:06   0:00 sleep 30s
root       599  0.0  0.0   7456  4120 pts/0    Ss   13:06   0:00 sh
root       604  0.0  0.0  12796  3100 pts/0    R+   13:06   0:00 ps aux
```

Log output of latest:

```bash 
Sat May 11 13:05:35 2019 OpenVPN 2.4.7 [git:makepkg/2b8aec62d5db2c17+] x86_64-pc-linux-gnu [SSL (OpenSSL)] [LZO] [LZ4] [EPOLL] [PKCS11] [MH/PKTINFO] [AEAD] built on Feb 19 2019
Sat May 11 13:05:35 2019 library versions: OpenSSL 1.1.1b 26 Feb 2019, LZO 2.10

2019-05-11 13:05:35,252 DEBG 'start-script' stdout output:
[info] OpenVPN started

2019-05-11 13:05:35,252 DEBG 'start-script' stdout output:
Sat May 11 13:05:35 2019 TCP/UDP: Preserving recently used remote address: [AF_INET]185.220.70.131:1198
Sat May 11 13:05:35 2019 UDP link local: (not bound)
Sat May 11 13:05:35 2019 UDP link remote: [AF_INET]185.220.70.131:1198

2019-05-11 13:05:35,311 DEBG 'start-script' stdout output:
Sat May 11 13:05:35 2019 [59e9f91df0f5ef85e9a75e23770dc838] Peer Connection Initiated with [AF_INET]185.220.70.131:1198

2019-05-11 13:05:36,478 DEBG 'start-script' stdout output:
Sat May 11 13:05:36 2019 TUN/TAP device tun0 opened
Sat May 11 13:05:36 2019 /usr/bin/ip link set dev tun0 up mtu 1500

2019-05-11 13:05:36,479 DEBG 'start-script' stdout output:
Sat May 11 13:05:36 2019 /usr/bin/ip addr add dev tun0 local 10.71.10.6 peer 10.71.10.5

2019-05-11 13:05:36,483 DEBG 'start-script' stdout output:
Sat May 11 13:05:36 2019 Initialization Sequence Completed

2019-05-11 13:05:36,501 DEBG fd 17 closed, stopped monitoring <POutputDispatcher at 22817772100784 for <Subprocess at 22817772028056 with name privoxy-script in state RUNNING> (stdout)>
2019-05-11 13:05:36,501 DEBG fd 21 closed, stopped monitoring <POutputDispatcher at 22817772101504 for <Subprocess at 22817772028056 with name privoxy-script in state RUNNING> (stderr)>
2019-05-11 13:05:36,501 INFO exited: privoxy-script (exit status 0; expected)
2019-05-11 13:05:36,502 DEBG received SIGCLD indicating a child quit
2019-05-11 13:05:36,517 DEBG 'nzbget-script' stdout output:
[info] Nzbget not running
[info] Attempting to start Nzbget...

2019-05-11 13:05:36,552 DEBG 'nzbget-script' stdout output:
[info] Nzbget started
```

**Comment by jarlave** [comment]
you could replace the nzbget.sh in /home/nobody/ with my little fix from my PR for the time being https://github.com/jarlave/docker-nzbgetvpn/blob/patch-1/run/nobody/nzbget.sh

**Comment by jshridha** [comment]
Fixed in b58300d01d44fa208030a3f2f6bd322f4799f351
